x/concentrated-liquidity/swaps_test.go

```diff
@@ -534,7 +504,8 @@ func (s *KeeperTestSuite) TestCalcAndSwapOutAmtGivenIn() {
 			test.addPositions(s.Ctx, pool.GetId())
 
 			// perform calc
-			tokenIn, tokenOut, updatedTick, updatedLiquidity, updatedSqrtPrice, err := s.App.ConcentratedLiquidityKeeper.CalcOutAmtGivenInInternal(
+			// TODO: Add sqrtPrice check
```

Comment:
Let's make an issue for this please

Reply:
https://github.com/osmosis-labs/osmosis/issues/3989